admin_guide/out_of_resource_handling.adoc

```diff
@@ -64,8 +68,7 @@ Each threshold is of the following form:
 <eviction_signal><operator><quantity>
 ----
 
-* Valid `eviction-signal` tokens as defined in
-xref:out-of-resource-eviction-signals[Eviction Signals].
+* Valid `eviction-signal` tokens as defined by eviction signals.
 * Valid `operator` tokens are `<`.
 * Valid `quantity` tokens must match the quantity representation used by
 Kubernetes.
```

Comment:
OK. I get you. I made a link out to the docs above, but you're right, it might be a good idea for us on docs to doc this at some point.